### ISSUE TITLE
There is no script for updating an existing gene panel.

### ISSUE THREAD

**Issue by Luke-Sikina** [cBioPortal/cbioportal#7286]
[The documentation](https://docs.cbioportal.org/5.1-data-loading/data-loading/import-gene-panels#reimport-existing-gene-panel) suggests that I delete the existing gene panel, then reimport the gene panel, then reimport the corresponding studies. That's a lot when you just want to add a gene.

We should make a script that modifies the existing gene panel by adding to the `gene_panel_list` table.

I think this is a good first issue, so I am tagging it as such.

**Comment by pieterlukasse** [comment]
I agree this can be made easier. At the same time, it should be made clear to users that this is the way to do it _only_ in case of fixes to the panel list. This is not the way to *update* a panel to a new version of the panel. E.g. there is a "panel A" for 200 genes and some studies have been done using this panel and loaded in cBioPortal. Some time later, the "same" panel is updated to include 10 more genes. In this case, this is actually a new panel, and should be loaded separately as "panel A v2" for example. Most likely, the same studies will have updated data, generated using the new panel. This new data needs to be re-imported. 
Bottom line: I agree with the proposed simplification of the process, but it would be good to at least warn the user if the gene panel he is updating already has some studies linked to it.
